tests/sentry/rules/filters/test_age_comparison.py

```diff
@@ -11,10 +10,8 @@
 class AgeComparisonFilterTest(RuleTestCase):
     rule_cls = AgeComparisonFilter
 
-    @mock.patch("django.utils.timezone.now")
-    def test_older_applies_correctly(self, now):
-        now.return_value = datetime(2020, 8, 1, 0, 0, 0, 0, tzinfo=pytz.utc)
-
+    @freeze_time()
```

Comment:
Let's freeze time here and below to the date we had specified as well. Just want to make sure we don't inadvertently make these tests flakey

Reply:
Oh woops, good catch.

Reply:
Ahh, didn't realise it was using `store_event`. In that case this might be fine. If you want to be a bit safer, one option here is to take the current date and just set the hour/minute/second etc to 0 to match what the old freeze was doing

Reply:
Whew that sent me down a bit of a rabbithole, but I think it should be good now.